user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Deleting_custom_apps.md

```diff
@@ -4,18 +4,25 @@ uid: Deleting_custom_apps
 
 # Deleting a low-code application
 
-> [!NOTE]
-> To view, add, edit, delete, or publish low-code applications, you need to have the necessary user permissions under [Modules > User-definable apps](xref:DataMiner_user_permissions#modules--user-definable-apps) as well as the [DataMiner web apps](xref:DataMiner_user_permissions#general--dataminer-web-apps--dataminer-cube-mobile-access) user permission.
+You can delete a low-code app from within the app or from the DataMiner landing page.
+
+From the DataMiner **landing page** (supported from DataMiner 10.3.0 [CU10]/10.4.0 onwards<!-- RN 37830 -->):
+
+1. Hover the mouse pointer over the app icon until you see the ellipsis ("...") button.
 
-To delete an existing low-code application:
+1. Click the ellipsis button and select *Delete*.
 
-1. Open the application from the DataMiner landing page (see [Accessing the Low-Code Apps module](xref:Accessing_custom_apps)).
+1. Click *Confirm*.
+
+From **within an application**:
 
 1. Click the user icon in the top-right corner and select *Edit*.
 
 1. Click the garbage can icon next to the user icon.
```

Comment:
In which version has this been moved? Because I'm looking at a test DMA using 10.4.1, and there it's still as described in the PR:
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/94605575/0511cf73-0988-401d-9d24-77b5cfd03cb7)